live/global.ini

```diff
@@ -22891,113 +22891,113 @@ PU_BANISTER_MG_Celebrate_IG_003_LoveIt=Ich liebe es!
+PU_BANISTER_MG_CommMsg_JobOffer_IG_001_HelloItsMe=Hallo. Ich bin's. Tessa. Junior-Kartographin. Komm vorbei, wenn Du kannst. Ich habe etwas für dich.
+PU_BANISTER_MG_CommMsg_JobOffer_IG_002_HiItsTessa=Hi, hier ist Tessa. Ich hatte gehofft, Du hättest vielleicht eine Sekunde Zeit, um vorbeizukommen. Ich habe etwas, das Dich interessieren könnte.
+PU_BANISTER_MG_CommMsg_JobOffer_IG_003_HeyThereTessa=Hallo zusammen. Tessa hier. Ich hoffe, ich störe Dich nicht zu sehr. Mir ist etwas eingefallen und ich dachte dabei an Dich. Wäre toll, wenn Du vorbeikommen könntest.
+PU_BANISTER_MG_CommMsg_JobOffer_IG_004_HeyItsTessa=Hey, hier ist Tessa. Habe etwas, wofür ich dich gebrauchen könnte. Nun ja, Gebrauch ist ein schlechtes Wort. Wie auch immer, wenn Du Zeit hast, komm vorbei.
+PU_BANISTER_MG_CommMsg_JobOffer_IG_005_HeyThereGuess=Hey. Rate wer hier ist. Ich bin auf etwas gestoßen, das Dich meiner Meinung nach interessieren wird. Komm vorbei, wenn Du kannst.
+PU_BANISTER_MG_CommMsg_ReturnProperty_FailSecurity_IG_001_ListenIDidnt=Hör mal, ich wollte nicht, dass es so weit kommt, aber es gibt einen Haftbefehl gegen Dich bezüglich des Eigentums, das Du gestohlen haben. Entschuldigung.
+PU_BANISTER_MG_CommMsg_ReturnProperty_FailSecurity_IG_002_WellYouDidnt=Nun, Du hast das Eigentum nicht zurückgegeben und jetzt mussten wir es als gestohlen melden. Ich denke, das macht Dich technisch gesehen zu einem Dieb, also solltest Du dich vielleicht darum kümmern.
+PU_BANISTER_MG_CommMsg_ReturnProperty_IG_001_HeyEhYou=Hey, eh, Du hast noch etwas Eigentum von mir. Du musst es mir zurückgeben.
+PU_BANISTER_MG_CommMsg_ReturnProperty_IG_002_LooksLikeYou=Sieht so aus, als hättest Du noch etwas von meinem Eigentum. Es wäre großartig, wenn Du es zurückbringen könnest.
+PU_BANISTER_MG_CommMsg_ReturnProperty_LastWarning_IG_001_HeyIfYou=Hey, wenn Du das Eigentum der ICC nicht zurückgibst, muss ich dich melden.
+PU_BANISTER_MG_CommMsg_ReturnProperty_LastWarning_IG_002_JustAWord=Nur ein Wort der Warnung. Die ICC wird Dich anzeigen, wenn Du ihr Eigentum nicht zurückgibst.
+PU_BANISTER_MG_CommMsg_UnderAttack_IG_001_HeyEhLong=Hey. Um es kurz zu machen: Ich werde gerade angegriffen und brauche wirklich Hilfe. Bitte beeil dich. Und bring ein paar Waffen mit. Viele Waffen.
+PU_BANISTER_MG_CommMsg_UnderAttack_IG_002_MaydayThisIs=Mayday! Hier ist Tessa Banister. Ich werde angegriffen. Ich brauche dringend  Hilfe. Mayday! SOS! Notfall! Außerdem ist es wirklich seltsam, so etwas zu sagen.
+PU_BANISTER_MG_CommMsg_UnderAttack_IG_003_HeyEhHope=Hey. Ich hoffe, Du bist nicht beschäftigt, denn gerade schießen ein paar Schiffe auf mich und es wäre großartig, wenn Du sie aufhalten könntest. Danke! Beeil dich!
+PU_BANISTER_MG_EncourageDecision_IG_001_SeemsLikeA=Scheint gut zu Dir zu passen. Was denkst Du?
+PU_BANISTER_MG_EncourageDecision_IG_002_YouThinkIts=Denkst Du, das wäre etwas für dich?
```

Comment:
Vorschlag: `Glaubst du, das ist etwas, was du tun möchtest?`

Engl. `You think it's something you want to do?`

Passt auch besser zum Rest